PIL/Image.py

```diff
@@ -2358,6 +2466,7 @@ def frombuffer(mode, size, data, decoder_name="raw", *args):
 
 
 def fromarray(obj, mode=None):
+    # type: (object, Optional[Mode]) -> Image
```

Comment:
This could be reasonable - but that depends:
a) np is not a given prerequisite for PIL[low], right?
b) should it be a prerequisite for type-checking? (checking compatibility)
c) if b, what version?